### PR TITLE
[FEATURE] Workspace: Add getModules method / Specification: Add getId method

### PR DESCRIPTION
**Pull request by matz3**
Required for https://github.com/SAP/ui5-project/pull/589

### CODE REVIEW

**Review by flovogt — COMMENTED** [review]
LGTM. Waiting for review by @KlattG

File: lib/graph/Workspace.js
Comment (matz3) [reply lib/graph/Workspace.js:87]: No, as it will be casted to a string by localeCompare. I added the first `String()` to make sure that localeCompare can be called. But we could also omit that as we know that we only use strings as keys.

**Review by KlattG — CHANGES_REQUESTED** [review]

File: lib/specifications/Specification.js
Comment (KlattG) [inline lib/specifications/Specification.js:198]:
```suggestion
	 * Gets the type of this specification,
```

**Review by KlattG — CHANGES_REQUESTED** [review]
Typo

File: lib/specifications/Specification.js
Comment (matz3) [reply lib/specifications/Specification.js]: Oops. Thanks.

**Review by KlattG — APPROVED** [review]
LGTM

**Pull request merged**
